GitHub_Login.html

```diff
@@ -1,86 +1,114 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=>">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=>" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>GitHub Login</title>
     <style>
-        body{
-            font-family: Arial, Helvetica, sans-serif ;
-            
-        }
-        .paragraph {
-            width: 260px;
-            font-size: 11.5px;
-            color: rgb(122, 123, 126);
-            margin-left: 5px;
-            line-height: 18px;
-        }
-        .inputs{
-            width: 260px;
-            height: 43px;
-            border-radius: 5px;
-            border: groove;
-            margin-top: 8px;
-            
-        }
-        #lastInput{
-            margin-bottom: -5px;
-        }
-        .labelClass{
-            font-size: 13px; 
-            margin-top: -100px;
-            color: rgb(92, 93, 95);
-        }
-        .button{
-            background-color: rgb(42, 189, 72);
-            width: 260px;
-            height: 70px;
-            color: white;
-            font-size: 16px;
-            font-family: Verdana, Geneva, sans-serif;
-            border-radius: 5px;
-            text-align: middle;
-            margin-bottom: -5px;
-            
-        }
-        #tab1{
-            margin-left: 12px;
-        }
-        #tab2{
-            margin-left: 10px;
-        }
-        body>section>p>a{
-            text-decoration: none;
-            
-        }
-        body>section{
-            font-size: 12px;
-        }
-        body{
-            margin-left: 15px;
-        }
-        
+      body {
+        font-family: Arial, Helvetica, sans-serif;
+      }
+      .paragraph {
+        width: 260px;
+        font-size: 11.5px;
+        color: rgb(122, 123, 126);
+        margin-left: 5px;
+        line-height: 18px;
+      }
+      .inputs {
+        width: 260px;
+        height: 43px;
+        border-radius: 5px;
+        border: groove;
+        margin-top: 8px;
+      }
+      #lastInput {
+        margin-bottom: -5px;
+      }
+      .labelClass {
+        font-size: 13px;
+        margin-top: -100px;
+        color: rgb(92, 93, 95);
+      }
+      .button {
+        background-color: rgb(42, 189, 72);
+        width: 260px;
+        height: 70px;
+        color: white;
+        font-size: 16px;
+        font-family: Verdana, Geneva, sans-serif;
```

Comment:
On rare occasions we have two fonts, buttons aren't the case tho 🙂 

In 99% of the cases, fonts are only set on the body tag